vendor/aws-vault/Makefile

```diff
@@ -0,0 +1,11 @@
+export DOWNLOAD_URL ?= $(REPO_URL)/releases/download/v$(VERSION)/$(NAME)-$(OS)-$(ARCH)
+
+include ../../tasks/Makefile.package
+include ../../tasks/Makefile.apk
+
+# Package details
+export VENDOR ?= 99designs
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
```

Comment:
So this is where I think we need to create a new template for `APKBUILD.github-tarballs`

Reply:
In that file, we need to extract the tgz and put the binary in place.

Reply:
https://github.com/cloudposse/packages/pull/70/files#diff-0669ec21c0d3ecac6c1922bdc48133f7R34

This is where we would need to decompress the tarball.

So the reason for the `install` target is for backwards compatibility with the original way of managing packages. This is so it's compatible with "native" installs on OSX (for example), which don't use alpine.

Maybe we should find some way to generalize it.